src/components/dataset-details/dataset-details.js

```diff
@@ -14,7 +15,7 @@ const DatasetDetails = ({ dataset, downloadUrl }) => {
       <div className='dataset-details-header'>
         <div data-testid='dataset-title' className='name'>{dataset.title}</div>
         <div className='buttons'>
-          <DownloadButton url={downloadUrl} />
+          <DownloadButton url={downloadUrl} format={getDefaultFormat(dataset)} filename={dataset.id} />
```

Comment:
Did we always use the dataset ID as the filename?  I thought we had made a point of not exposing the actual dataset ID to our users.

Reply:
Looks like it does... wonder if we should continue to do that.  🤔

Reply:
To be fair, every dataset details page returns dataset details including the dataset id... so id isn't even really obscured. But yeah, it'd be nice to use system name so it's not in the user's face.